python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/vxlan_interface.schema.yml

```
@@ -154,7 +154,29 @@ keys:
                         - str
                     multicast_group:
                       type: str
-                      description: IP Multicast Group Address.
+                      description: Default IP Multicast Group Address for the VRF.
+                    multicast_group_encap_range:
+                      type: str
+                      description: |-
+                        N:M mapping, the overlay groups in the VRF are mapped to one of
+                        the underlay groups defined within the encap range.
+                        The format is X.X.X.X/YY.
```

Maybe a bit more clarification to help users?
```suggestion
                        The format is X.X.X.X/YY (IPv4 multicast group with prefix length between 19 and 32).
                        Default multicast group `vxlan_interface.vxlan1.vxlan.vrfs.[].multicast_group` is required for this feature to work.
                        See the TOI at https://www.arista.com/en/support/toi/eos-4-29-1f/16546-multicast-evpn-irb-multiple-underlay-groups
```